server/middleware.json

```diff
@@ -17,6 +17,5 @@
     "loopback#urlNotFound": {}
   },
   "final:after": {
-    "errorhandler": {}
```

Comment:
That was used initially, then we agreed to use `errorhandler` instead. .. reverting back now. Fixed.